.github/workflows/storybook.yml

```
@@ -0,0 +1,26 @@
+name: Storybook
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: 'npm'
+          cache-dependency-path: package-lock.json
+        
+      - name: Deploy storybook to Github Pages
+        run: |
+          npm install --legacy-peer-deps
+          # Forcing legacy peer dep behavior, since we want to use a version of React 
+          # that is newer than the one Storybook expects.
+          npm run deploy-storybook -- --ci
```

I think the flag is redundant here, since it's already in the package.json script.
```suggestion
          npm run deploy-storybook
```